code/modules/admin/verbs/SDQL2/SDQL_2.dm

```diff
@@ -389,6 +396,12 @@
 	else if(copytext(expression[i], 1, 2) in list("'", "\""))
 		val = copytext(expression[i], 2, length(expression[i]))
 
+	else if(expression[i] == "\[")
+		var/list/expressions_list = expression[++i]
+		val = list()
+		for(var/list/expression_list in expressions_list)
+			val += SDQL_expression(object, expression_list)
```

Comment:
It just occurred to me that nested lists probably don't work.

Make this `val[++val.len] =` instead.